homeassistant/components/nest/translations/en.json

```diff
@@ -1,52 +1,59 @@
 {
```

Comment:
FYI I just did a hard copy of strings.json to en.json to ensure it is updated, since it looked stale.

Let me know if its simpler to just ignore this and let it get auto updated.

Reply:
It's ok during development to run the translations develop script to get an English translation based on strings.json.

The translation files are updated via Lokalise so changes will be overwritten by our infrastructure anyway.

Reply:
Thanks, I forgot about `python3 -m script.translations develop`.

I've run the script so it should reduce the diff.